Gemfile.lock

```diff
@@ -167,6 +169,7 @@ GEM
     zeitwerk (2.4.2)
 
 PLATFORMS
+  x86_64-darwin-19
```

Comment:
This probably shouldn't be here. Not sure though

Reply:
This went automatically after the bundle install or bundle update (don't remember each one right now). But I will remove it.